solana/modules/token_bridge/program/src/messages.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(clippy::manual_memcpy)]
```

Comment:
Same here.  Probably better to put this next to the call site so that it can be removed if we ever change it.